Scripts/Items/Consumables/EodonPotions.cs

```diff
@@ -802,30 +802,61 @@ public override void Deserialize(GenericReader reader)
         }
     }
 
-    public class BlueCornStock : Item
```

Comment:
@Line 807 that you added: public override int LabelNumber { get { return 1035639; } } // corn stalk

Also, backwards compatibility is needed not just for code for spawners, etc. but also for shards that have existing blue corn items spawned, otherwise they will be forced to delete them. (Unless this is an all new item, or something that respawns on the ground and cannot be picked up by players)

Edit: Okay, I've looked at this in game, and it does not appear to be something players can pick up, so you can probably change the name without much issue(The admins of shards that have them spawned will still be prompted to remove them from game, though), however you need to change it to CornStalk rather than CornStock. No point in changing it from one erroneous name to another.